lively.ide/js/workspace.js

```diff
@@ -27,6 +27,7 @@ export default class Workspace extends Window {
             name: 'editor',
             lineHeight: 1.4,
             readOnly: false,
+            needsDocument: true,
```

Comment:
We can of course say that we always want to set this explicitly, but if you did not change that (I have not yet read all changes), this is not necessary. Since this morph has `readOnly == false`, its text will always be backed by a document, as we need that for interactive editing capabilities.
The flag was introduced for cases where we do not want to set `readOnly = true`, but still need some capabilities that the document provides us. The prime example are interactive trees like the Inspector or the scene graph, where we do not want to interactively edit the content.

I actually remember, that we talked about this in the past and you were not super happy with the solution and wanted to have a third user exposed flag? But I am not sure anymore, sorry :(

Reply:
This was a temporary change, that fixed the initial problem with workspaces crashing the world when they are deserialized.